meetings/2023-09-06.md

```diff
@@ -0,0 +1,164 @@
+
+* eP: Rigo mentioned something about test suite. Does the charter already address or do we need to do something special?
+* RW: Test suite is double licenced, you don't need to do anything. If you make claims of conformance you cannot alter. 
+* eP: For test suite it is better not to use MIT but the one recommended by W3C. 
+* RW: Not very different. For the test suite, for claims of conformance or intend to use it for claims of conformance, you cannot take altered code, because then your claims would be false. Copyright is an additional kind of helper, not main target. 
+* WT: The sentences in the charter about this is exactly what W3C has about double licensing, so we don't deviate from that. 
+* SC: RW, in the license section we have to "note that". Do you recommend that we keep that text in or remove? 
+* RW: First of all you say WG instead of CG which is wrong. I think per W3C process this is a community process so normal process doesn't apply. I don't thin kit makes a legal different but would look nicer if you note that artifacts that become part of this group's TR will fall under W3C software license.
+* WT: The notes are meant to indicate that independent of the license we use, these items will be under a WG's license. 
+* RW: You can leave that to the WG. I would keep the paragraph but make it so that it reads as "the intention is to..." because whether it happens or not we will decide later.
+* SC: Can we add a revision to that before we merge? WT. 
+* WT: Yes. 
+* eP: In the next 5 minutes?
+* SC: Yes.
+* RW: You don't have to use test suite license for tests. 
+* SC: Can we draft in the minutes now? 
+* RG: Can I ask RW to put the recommendations somewhere here we can follow?
+* RW: I can create a pull request.
+* RG: Appreciate it.
+* WT: "Note that the intention of this group is to propose its documents as Technical Reports to a Working Group, where the published documents will fall under the [W3C Document License]."
+* SC: Works for me.
+* RW: Completely fine.
+* WT: "Note that the intention of this group is to propose its software artifacts and code snippets included in documents to a Working Group's Technical Reports, where they will fall under the [W3C Software License] without notification obligation."
+* RW: That's fine, exactly what I wanted.
+* WT: "In accordance with the CLA, the group licenses all its software artifacts under the [W3C Software license], but without notification obligation."
+* RW: Fine. The important part is distinction between document that has rules and software artifacts. 
+* eP: Software artifacts, only snippets or something other than test suites? 
+* RW: Here we talk about the rules, understanding software snippets that are written in the spec: you have a rule "xyz, software must do abc, here is an example". When you have other software you want to put under W3C you have a free choice of license. My favorite is Apache because it allows commercial usage but offers patent protection. 
```

Comment:
```suggestion
* eP: Software artifacts: only snippets, or something other than test suites? 
* RW: Here we talk about the rules, understanding software snippets that are written in the spec: you have a rule "xyz, software must do abc, here is an example". When you have other software you want to put under W3C, you have a free choice of license. My favorite is Apache, because it allows commercial usage but offers patent protection. 
```